pulseguardian/model/pulse_user.py

```diff
@@ -27,16 +27,19 @@ class PulseUser(Base):
         Queue, backref='owner', cascade='save-update, merge, delete')
 
     @staticmethod
-    def new_user(username, password='', owner=None, management_api=None):
+    def new_user(username, password='', owner=None, skip_management=False):
```

Comment:
I'm not crazy about this variable name. Although I admit the current behaviour when management_api is None is not terribly obvious, this is an opportunity to fix it. :) Maybe "create_rabbitmq_user=True" (and reversing the logic below)? Or something like that?